### DISCUSSION TITLE
Archived date not shown for repository

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#146107] [Mobile] [answered]
GitHub Mobile beta for Android

Open https://github.com/actions-importer/preview
See:
![Screenshot_20241202-202310.png](https://github.com/user-attachments/assets/b7a4b6c1-03fb-417d-880d-230628aea796)

Compare in browser:
![Screenshot_20241202-202657.png](https://github.com/user-attachments/assets/cb43b323-3f21-4e8b-9316-bec0d0fd5eed)

**Comment by stevepopovich** [comment, marked as answer]
Hello @jsoref thanks for the request. I have passed it onto the team.
